iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -752,6 +752,33 @@ void ConvertToGPUPass::runOnOperation() {
                   MapLinalgOpToGlobalInvocationId<linalg::GenericOp>,
                   MapLinalgOpToGlobalInvocationId<linalg::IndexedGenericOp>,
                   MapLinalgOpToLocalInvocationId<linalg::ConvOp>,
+                  MapLinalgOpToLocalInvocationId<linalg::TfConvNWCOp>,
```

Comment:
Do we need all these cases here? Unless we have tests for them, I think it's better to just enable those we have good support and tests and let the rest fail loudly. Otherwise we may hit issues later due to no testing and we assume it works.

Reply:
Yes, we need them. I will add some tests.

Reply:
Still need to add tests for conv1d/3d

Reply:
added end-to-end tests for conv1d/3d